docs/rules/array-callback-return.md

```diff
@@ -20,14 +20,17 @@ This rule finds callback functions of the following methods, then checks usage o
 * [`Array.prototype.every`](https://www.ecma-international.org/ecma-262/6.0/#sec-array.prototype.every)
 * [`Array.prototype.filter`](https://www.ecma-international.org/ecma-262/6.0/#sec-array.prototype.filter)
 * [`Array.prototype.find`](https://www.ecma-international.org/ecma-262/6.0/#sec-array.prototype.find)
-* [`Array.prototype.findIndex`](https://www.ecma-international.org/ecma-262/6.0/#sec-array.prototype.findIndex )
+* [`Array.prototype.findIndex`](https://www.ecma-international.org/ecma-262/6.0/#sec-array.prototype.findindex )
```

Comment:
as a side note, all of these links shouldn't be to the always-obsolete snapshots on ecma-international.org, but rather to the only correct version of the spec, https://tc39.es/ecma262/

Reply:
I'd like to fix the links in another PR, as two of them are partially broken (wrong `#`).

I understand that [https://tc39.es/ecma262/](https://tc39.es/ecma262/) link represents the actual version much more than a draft, but it's officially still a draft? Also, it has the Draft word in the title, so I'm not sure is it OK to use this link, or just fix the links to ES2015. Or maybe change all links to ES2019.

Reply:
It’s a living standard; whatever’s on github is always the latest version.